test/export/test_export.py

```diff
@@ -3897,9 +3912,10 @@ def forward(self, x):
         )
 
     def test_sym_stack_trace(self):
+        # TODO(avik): update this test with torch._check*
         class Foo(torch.nn.Module):
             def forward(self, x, y):
-                y = torch._constrain_as_size(y.item(), min=2)
+                y = torch.sym_constrain_range_for_size(y.item(), min=2)
```

Comment:
Do you know if `torch.sym_constrain_range_for_size` is a maintained API? I sort of forgot this API existed 😅 . I thought we wanted to move everything to `torch._check` / `torch._check_is_size`

Reply:
Not sure. Leaving the TODO for now, yeah I don't want to recommend it, but it exists and I don't know why. All I know is that `_constrain_as*` directly desugars into this.

Reply:
Ahh i think it is ok to delete it. It was for preserving the op in the aten graph so we can always end up re-constraining when we retrace.